app/Http/Controllers/Auth/VATSIMOAuthController.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+namespace App\Http\Controllers\Auth;
+
+use League\OAuth2\Client\Provider\Exception\IdentityProviderException;
+use League\OAuth2\Client\Provider\GenericProvider;
+
+class VATSIMOAuthController extends GenericProvider
```

Comment:
Is this a controller if it is not serving any HTTP requests?

Reply:
Nope! A provider...
3df10fb5